docs/source/tutorial/tutorial-authentication.mdx

```diff
@@ -134,19 +132,15 @@ private func enableSubmitButton(_ isEnabled: Bool) {
 }
 ```
 
-Next, add a simplified email validation to ensure that the String you'll be passing through to the server is an email address:
+Next, add basic email validation to help ensure that the String you'll be passing through to the server is an email address:
```

Comment:
If you're going to nuke the note about this not being a real email validator, we should say some thing like `...to the server could at least possibly be an email address:`